src/keys-api/keys-api.service.ts

```diff
@@ -39,19 +41,31 @@ export class KeysApiService {
     }
   }
 
-  public async getModulesList() {
-    const result = await this.fetch<SRModuleListResponse>('/v1/modules');
-    if (!result.data?.length || !result.elBlockSnapshot)
-      throw Error('Keys API not synced, please wait');
+  /**
+   *
+   * @param The /v1/keys/find API endpoint returns keys along with their duplicates
+   * @returns
+   */
+  public async findKeysEntires(pubkeys: string[]) {
```

Comment:
Looks like there is a convention for naming class methods that fetches data from KAPI  - they start with `get` and then follows with `what we are getting`. This method has a typo and starts with `find` and has strange `entries` in the name. I think it's better to rename it to `getKeysByPubkeys`.

Reply:
okay, will fix